radio/src/lua/api_model.cpp

```diff
@@ -1586,11 +1586,69 @@ static int luaModelResetSensor(lua_State *L)
   if (idx < MAX_TELEMETRY_SENSORS) {
     telemetryItems[idx].clear();
   }
-  
+
   lua_pushnil(L);
   return 1;
 }
+/*luadoc
+@function model.swashRingData(index, line,{parameters})
+
+Set heli swash parameters
+
+index = 0
```

Comment:
Not sure what this means ?

Reply:
Parameter renamed and explained.

Since this is basically an internal mix it follows the same format as a mix
index refereed to "which" mix (swash) and Line refereed to what line.

Since this is the only mix line for this item I simply described it so the user no option other than to use "0" for these 2 items.

I have tried changing the values to 1 & 2, from 0 they have no effect, however to eliminate any possible conflicts I suggest they be set to "0" in the descripton.

![doc changes](https://user-images.githubusercontent.com/20526716/125211148-6430c900-e272-11eb-95eb-34e8e7d09b2f.PNG)